packages/shared/src/types/feature-flags.ts

```diff
@@ -43,4 +43,5 @@ export enum FeatureFlagsKeysEnum {
   IS_CONTROLS_AUTOCOMPLETE_ENABLED = 'IS_CONTROLS_AUTOCOMPLETE_ENABLED',
   IS_USAGE_ALERTS_ENABLED = 'IS_USAGE_ALERTS_ENABLED',
   IS_NEW_DASHBOARD_ENABLED = 'IS_NEW_DASHBOARD_ENABLED',
+  IS_ND_DELAY_DIGEST_EMAIL_ENABLED = 'IS_ND_DELAY_DIGEST_EMAIL_ENABLED',
```

Comment:
It's not critical, just a suggestion. 🙃
If the ND means a new dashboard, I would add the full name or remove it all together.

Reply:
Correct. We have `IS_NEW_DASHBOARD_ENABLED` env variable so the prefix should be the same.

Reply:
I agree the naming could be better but I will not change it now - Its already in LD (can't be deleted just archived), its used elsewhere and its temporary + the description in LD states specifically what is it.